docs/sphinx/using/basics/basics.rst

```diff
@@ -0,0 +1,27 @@
+CUDA Quantum Basics
+*******************
+
+.. toctree::
+   :caption: Contents
+   :maxdepth: 2
+
+      What is a CUDA Quantum Kernel? <kernel_intro.rst>
+      Building your first CUDA Quantum Program <build_kernel.rst>
+      Running your first CUDA Quantum Program <run_kernel.rst>
+      Troubleshooting <troubleshooting.rst>
+
+..
+  Fix Me: Move to own file when ready to fill out section (Issue #1352)
+
+  Language Fundamentals
```

Comment:
Please remove this section completely. It is not written, so it should not be here. The documentation will be live update when we merge this change to main; there certainly shouldn't be a Todo visible in the docs, or any empty/half finished paragraphs.